astropy/coordinates/tests/test_earth.py

```diff
@@ -288,21 +286,34 @@ def test_repr_latex():
 
 @pytest.mark.remote_data
 def test_of_address():
+    # just a location
+    try:
+        loc = EarthLocation.of_address("New York, NY")
+    except NameResolveError as e:
+        # Google map API limit might surface even here in Travis CI.
+        pytest.xfail(str(e))
+    else:
+        assert quantity_allclose(loc.lat, 40.7128*u.degree)
+        assert quantity_allclose(loc.lon, -74.0059*u.degree)
+        assert np.allclose(loc.height.value, 0.)
+
+    # Put this one here as buffer to get around Google map API limit per sec.
```

Comment:
Maybe write in the comment that this raises `NameResolveError` no matter whether google does or does not impose its limit.

Reply:
@mhvk , I added clarification. If this looks good to you, please approve so it can be merged. Thanks!